### PR TITLE
Adding Leather Works support

### PR DESCRIPTION
**Pull request by Balint66**
Adding plugin for Leather Works mod, and updating forgegradle to be able to use curseforge maven API.

### CODE REVIEW

**Comment by KnightMiner** [comment]
Can you fix formatting to match the rest of the mod? Notably, brackets on the same line and sorted imports.

Also, I think I can guess by looking at the code, but mind giving a brief description of what this plugin does for the sake of documentation?

Side note, unless I'm calling that mods APIs I skip it as a build dependency so I don't need to always boot with it. Currently I just test using the libs folder, but I like the curseforge maven idea. Maybe keep it but leave leatherworks commented out?

**Comment by Balint66** [comment]
Of course. I will reformat my code and give a better description.

Oh, yes, sorry I forgot to comment out/delete the leather works dependenci line, my bad. 😅
I used it so I can search in it in my ide so I didn't have to switch between github and the ide.

I will change my code as soon as I can get to my computer.

**Review by KnightMiner — CHANGES_REQUESTED** [review]
I've left a couple more comments, but have one last question:

Is there any sort of time delay in Leather Works for making leather? I assume its barrel acts basically like my cauldrons, but want to make sure as otherwise it needs to be rebalanced (for example, with potions I only give you two potions instead of 3 to make up for instant crafting)

File: src/main/java/knightminer/inspirations/plugins/leatherworks/LeatherWorksPlugin.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        if(!Config.enableExtendedCauldron) {
+            return;
+        }
+
+        // get tannin fluid
+        Fluid tannin =  FluidRegistry.getFluid("leatherworks:tannin");
+
+        // register tannin and use it in recipes
+        if (tannin != null) {
+            InspirationsRegistry.addCauldronWater(tannin);
```
[the remark this answers is not in the document]
Comment (Balint66) [reply src/main/java/knightminer/inspirations/plugins/leatherworks/LeatherWorksPlugin.java]: Oh, for some reason I thought that it was for registering fluids. My bad.

**Comment by Balint66** [comment]
Leather Works use up the fluid instantly so it does not need any rebalance.

**Comment by KnightMiner** [comment]
Just wanted to update this to say the changes look good, just have not had a chance to test this yet. Will have time this weekend.

**Comment by Balint66** [comment]
I'm glad to hear it. 🙂

**Review by KnightMiner — APPROVED** [review]

**Comment by KnightMiner** [comment]
Just tested it and it looked great, seems their barrels already act a lot like cauldrons so it fits very well.

**Pull request merged**